### ISSUE TITLE
Add Livekit Webhooks Receiver Endpoint

### ISSUE THREAD

**Issue by chandansgowda** [AOSSIE-Org/Resonate-Backend#33]
We need an endpoint which receives web hooks from Livekit and performs required operation. One such use case is as follows.
1. Admin creates a room
2. Participants join the room
3. Admin loses internet connection or just force closes the app
4. Even if all participants leave the room, the room will be still active on Appwrite

This room has to be deleted as soon as the last participant leaves the room. Hence we can use a webhook to do that.

**Comment by chandansgowda** [comment]
This can be done only after deploying the project to a public web server. Will keep this on hold till then.

**Comment by xonx4l** [comment]
what I think solution could be  @chandansgowda . A feedback would be appreciated.

**Comment by chandansgowda** [comment]
Hey @xonx4l 
Thanks for the pull request. I am sorry to say that this issue is part of GSOC 2023 season and is not for other contributors. Also, there are a few changes in the PR that as to be made. Please feel free to look into other issues that are not labelled gsoc and get it assigned before working on it. You can also raise issues for feature requests and bug fixes. Thanks.

**Comment by Jyoticharan** [comment]
please assign me this issue

**Comment by ShamanthOP** [comment]
@chandansgowda I would like to work on this issue.
